src/js/quiz-component.js

```diff
@@ -0,0 +1,73 @@
+const quizComponent = (quizInfo) => {
+    const title = document.createElement("h2");
+    title.id = "title";
+    title.textContent = "Pop quiz!";
+  
+    const question = document.createElement("p");
+    question.id = "question";
+    question.textContent = quizInfo.question;
+
+    const submitBtn = document.createElement("button");
+    submitBtn.id = "submit-btn";
+    submitBtn.textContent = "Submit my answer!";
+    
+    // Here goes the function declared in the game logic module. Which handles the good and bad answers.
+    // submitBtn.addEventListener("click", answerHandler);
+
+    const quiz = document.createElement("div");
+    quiz.id = "quiz";
+    quiz.appendChild(title);
+  
+    const form = document.createElement("form");
+    form.id = "answers";
+
+    const nextChapterBtn = document.createElement("button");
+    nextChapterBtn.id = "next-chapter-btn";
+    nextChapterBtn.textContent = "Next chapter!";
+
+    // Here goes the function that links to the next chapter.
+    // nextChapterBtn.addEventListener("click", nextChapterFunction);
+
```

Comment:
Overall it looks good, but i think that after players chooses right answer, game logic should automatically generate chapter end component with info that quiz and chapter was successfully completed, so probably there is no need to generate next chapter btn.

Reply:
Oh yeah, that would probably be better, @DavidMatalik and @emdevs what do you think about it?

Reply:
I general I agree that there is no need for a nextBtn in quiz component. 

Be aware that chapterEnd takes in quiz-component as argument when chapterEnd is created. So how it is implemented at the moment there is no need for creating chapterEnd again. Just removing/hiding quiz component should be enough.

Reply:
Yeah its probably best not to include next chapter within this component, this ensures quiz component only has one responsibility. 

Initially my idea was to have quiz component trigger a change in the parent component (chapterEnd), so that the next button would appear on the chapterEnd component. Or some other visible change like hiding quiz component could work too. I'm not entirely sure how to implement this though